libraries/AP_HAL_ChibiOS/GPIO.cpp

```diff
@@ -511,6 +511,19 @@ bool GPIO::valid_pin(uint8_t pin) const
     return gpio_by_pin_num(pin) != nullptr;
 }
 
+// return servo channel associated with GPIO pin.  Returns true on success and fills in servo_ch argument
+bool GPIO::pin_to_servo_channel(uint8_t pin, uint8_t& servo_ch) const
+{
+    // search _gpio_tab for matching pin
+    for (uint8_t i=0; i<ARRAY_SIZE(_gpio_tab); i++) {
+        if (_gpio_tab[i].pin_num == pin) {
+            servo_ch = _gpio_tab[i].pwm_num;
```

Comment:
if pwm_num is zero then need to return false
if pwm_num is non-zero then need to add chan_offset if we have an IOMCU

Reply:
pwm_num of zero and chan_offset added.